### ISSUE TITLE
Broken form: external script

### ISSUE THREAD

**Issue by Frick** [prorenatalabs/prorenatalabs.github.io#1]
Besides the potential for exploit, you never know when things will change out from under you! In this case even more so since you've linked directly to `master` instead of a specific, tested commit hash. The script in question, [`form-submission-handler.js`](https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server/blob/master/form-submission-handler.js) actually changed from a [commit on July 29th](https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server/commit/ee8597094e000236532c8c909cec2add8fac2d4f) which replaced accessing the form by ID (just `gform`) with access by class (`form.gform`). Without further testing, the quick fix would be to pin back to the commit just prior to this one, otherwise bringing things up to date with the latest (and maybe serving it directly from the site?) might include further changes.

**Comment by bizarrowill** [comment]
Sounds good thanks for finding that! Feel free to make any changes needed or pin backs you think may work anytime. I was going to go back to the tutorial I used to create this and see what I could find. Just for some context none of this was designed as a real secured form beyond the google password, it was a short term, hacky way to use a google service to send an email for free upon a form submission. They didn't want a full server, database, etc until the budget called for it later. The entire thing could use a real secured approach with the scale it seems this is headed. That's all a bit beyond my scope for now this was the first form I've ever coded. This is what I used to create this form - https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server - I do have another version working still on a separate site I'm going to compare the scripts with and see if I can get a quick edit in later tonight. Please shoot an email to williamp.robertson@gmail.com if you have any questions about this and want real quick communication on it I'm much quicker there.  I just happened to see this here now that I've had a chance to start debugging this thing, the Caesars wifi was pretty garbage all weekend so I stayed off the laptop. If I get a working fix tonight I'll push it up.

**Comment by bizarrowill** [comment]
I think I got it figured out. In my other versions I had saved the original form-submission-handler.js locally and linked to it like the rest of my .js files. This was the first one I had ever built and must've not saved the local version by habit yet then. I never made any commits to that file after adding it to my other projects and the commit you linked to here on July 29th led me down the rabbit hole of realizing this site was still linked to the example blog's .js file. Thanks for posted that!  It's working in my local tests now with my saved .js file so I'm pushing it up here then copying things over to bluehost so it can go to the live site.
